editor/components/skip-to-selected-block/style.scss

```diff
@@ -0,0 +1,27 @@
+.skip-to-selected-block {
+	position: absolute;
+	top: -9999em;
+
+	&:focus {
+		height: auto;
+		width: auto;
+		display: block;
+		font-size: 14px;
+		font-weight: 600;
+		padding: 15px 23px 14px;
+		background: #f1f1f1;
+		color: $blue-wordpress;
+		line-height: normal;
+		box-shadow: 0 0 2px 2px rgba(0,0,0,.6);
+		text-decoration: none;
+		outline: none;
+		z-index: z-index( '.skip-to-selected-block' );
+	}
+}
+
+.edit-post-sidebar .skip-to-selected-block:focus {
```

Comment:
`edit-post` is a superset of (extends) the underlying `editor`, so there should not be awareness of that context from here. These styles, if needed, should be applied within `edit-post/components/sidebar/style.scss` (also better reflecting that they're specific to the edit-post sidebar, for someone looking for those styles).